### DISCUSSION TITLE
Controlling the length of the Whisper.cpp generated text segments

### DISCUSSION THREAD

**Discussion by GitGianluc** [URUWorks/TeroSubtitler#248] [Ideas]
It would be interesting to add to the control window of the "Audio to text..." command the possibility to specify the desired length of the automatically generated text lines by Whisper.cpp. By checking a radio button and entering the number of words per line, Whisper.cpp could thus take into account the `-ml %d` option flag (where `%d` is equal to the number of words per line).
More info --> [https://github.com/ggerganov/whisper.cpp?tab=readme-ov-file#controlling-the-length-of-the-generated-text-segments-experimental](https://github.com/ggerganov/whisper.cpp?tab=readme-ov-file#controlling-the-length-of-the-generated-text-segments-experimental)

**Comment by URUWorks** [comment]
Could be, you can also add it manually in the additional parameters by pressing "..." button next to the whisper.cpp engine.

**Comment by URUWorks** [comment]
v1.0.1.7
